.changeset/stupid-news-greet.md

```diff
@@ -0,0 +1,6 @@
+---
+'next-openai': patch
```

Comment:
```suggestion
```

Reply:
We only use changeset for the AI package, all the others are unpublished/unversioned